### DISCUSSION TITLE
What's the best way to install Rector?

### DISCUSSION THREAD

**Discussion by ruudk** [rectorphp/rector#6263] [Q&A] [answered]
I've been using `rector/rector` as a dependency and after upgrading to 0.10.12 I noticed:
```
ssch/rector-extension-installer: Extensions installed
> rector/rector-cakephp: installed
> rector/rector-doctrine: installed
> rector/rector-laravel: installed
> rector/rector-nette: installed
> rector/rector-phpunit: installed
> rector/rector-symfony: installed
```

I don't use `cakephp`, `laravel` and `nette` so I was wondering what the recommended way is to install rector nowadays? 

Should I just require `rector/rector-symfony, rector/rector-phpunit, rector/rector-doctrine, rector-extension-installer`?

**Comment by TomasVotruba** [comment]
Hi,

see this issue with similar question: https://github.com/rectorphp/rector/issues/5976

Personaly I would install `rector/rector-prefixed` that is the most isolated version with least potential conflicts :)
Maybe the only one for the future (dicussed in the issue too)

**Reply by josh091317** [reply]
I'm new to programming.  Can you write a step-by-step instructions on hw to install and run rector?

**Reply by TomasVotruba** [reply]
See https://getrector.org/documentation

**Comment by TomasVotruba** [comment, marked as answer]
The best way to install Rector is here: 
https://getrector.org/documentation
